services/dynamic-sidecar/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+All notable changes to this service will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
```

Comment:
MINOR: the square brackets in the markdown where you took this are to provide links but you have none here.

e.g
```markdown

[Unreleased]: https://github.com/olivierlacan/keep-a-changelog/compare/v1.0.0...HEAD
[1.0.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.3.0...v1.0.0
[0.3.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.2.0...v0.3.0

```

Reply:
It is a bit hard to add tags to the changelog, as the tags are referred to the entire project and not this service.